### PR TITLE
v1 of glossary added

### PR DESCRIPTION
**Pull request by jesseyuen**
We're creating a glossary of terms to help standardise the way we refer to our service and to assist with onboarding new tem members. 

This is the v1 of the glossary and it will be updated shortly with edits and new terms.

### CODE REVIEW

**Comment by paulrobertlloyd** [comment]
Should this not exist at the top level somewhere? I think a lot of these terms cross service boundaries.

**Comment by timabell** [comment]
Is there a nicer way of formatting the raw data? I don't fancy editing that much

**Comment by paulrobertlloyd** [comment]
@timabell We could use a straight-forward definition list, which can be done as follows in Markdown:

```
Accredited body (Accredited provider)
: Able to award QTS status (Accredited by DFE - the body does not accredit anyone).

Allocation
: Permission to run limited courses.

…
```

**Comment by jesseyuen** [comment]
> Should this not exist at the top level somewhere? I think a lot of these terms cross service boundaries.

Deffo something to consider @paulrobertlloyd  - maybe it's needs extending to cover Apply Terminology too?

**Comment by timabell** [comment]
review app https://deploy-preview-74--bat-design-history.netlify.com/find-teacher-training/glossary-of-terms-2020

**Comment by paulrobertlloyd** [comment]
Using a straight-forward definition list:

Screenshot 2020-02-25 at 12 45 31

Depends if we want the scalability of terms along the left of the page? We could also indent definition descriptions (`dd`) within `.app-prose` which would accomplish the same.

**Comment by jesseyuen** [comment]
> Using a straight-forward definition list:
> 
> Screenshot 2020-02-25 at 12 45 31
> 
> Depends if we want the scalability of terms along the left of the page? We could also indent definition descriptions (`dd`) within `.app-prose` which would accomplish the same.

_Potentially_ a teeny but less readable but far more maintanable. I missed a bracket or 4 doing this page it was annoying to fix. I'm happy to take this route personally.

**Comment by paulrobertlloyd** [comment]
Also, small nit, I think it’s `BaT`, not `BAT`.

**Comment by paulrobertlloyd** [comment]
…although that might just be me!

**Comment by fofr** [comment]
If we stick with the summary list it should be full width.

**Comment by fofr** [comment]
I don't think this document should live within Find, but should be at the same level as the sitemap. And linked to from footer.

**Comment by jesseyuen** [comment]
Link moved to footer and summary list styling for ease with maintainability. The next PR will look at the the other glossaries being compliled by the Apply team and adding additional items to the Design History.

File: app/_layouts/base.njk [hunk 1]
```diff
@@ -43,6 +43,9 @@
         }, {
           text: "Wiki",
           href: "https://dfedigital.atlassian.net/wiki/spaces/BaT/overview"
+        }, {
+          text: "Glossary",
+          href: "/glossary-of-terms"
```
Comment (paulrobertlloyd) [inline app/_layouts/base.njk]: This link is wrong. Should be `/glossary`

Comment (jesseyuen) [reply app/_layouts/base.njk]: Updated 😬

File: app/glossary-of-terms.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
```
Comment (fofr) [inline app/glossary-of-terms.md]: Can we rename the file to `glossary.md` too?

Comment (jesseyuen) [reply app/glossary-of-terms.md]: Done

**Review by fofr — APPROVED** [review]

**Pull request merged**